src/main/archetype/ui.frontend.react.forms.af/src/Form.tsx

```diff
@@ -42,13 +43,23 @@ const Form = (props: any) => {
             setForm(JSON.stringify(json.afModelDefinition))
         }
     }
+    const onSubmit= (action: Action) => {
+      const state = action.target.getState();
+      const thankyouMessage = state?.properties?.['afs:thankyouMessage'];
+      const thankyouPage = state?.properties?.['afs:thankyouPage'];
+      if(thankyouPage){
+        window.open(thankyouPage, '_blank');
```

Comment:
Instead of opening a new window, we should change the URL in the same window

Reply:
changed